karma.config.js

```diff
@@ -197,6 +225,81 @@ if (argv.includeCoverage) {
   plugins.push(require.resolve('karma-coverage-istanbul-reporter'));
 }
 
+// Create BrowserStack config when browsers CLI `browsers` param has `browserstack`
+if (argv.browserstack.length && !argv.watch) {
```

Comment:
Should we support to run without specific any browsers? For example exec command `npm run test -- --browserstack` then it will auto pick default browsers as same as ci being.